core/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
```

Comment:
I don't see this is exist or be used anywhere in the codebase, please confirm what this file is for? What is the purpose of this?